documents/operating.md

```diff
@@ -52,14 +52,202 @@ SECTION 4: 💵💵💵 MONEY 💵💵💵
+meeting chair may select alternative emoji to represent positive and negative votes.
+
+If there is no support for emoji or appropriate effects the Corporate Overlords must
+instead fabricate physical emoji on sticks for both positive and negative votes.  If an
+overlord is unable to fabricate an emoji, they must draw a shaded thumbs up and a
+thumbs down on their hand using a marker or pen.  If they don't have hands or a means
+to draw, they can work with the meeting chair to determine an appropriate alternative.
+
+A verbal vote may only be taken if the above methods are not reasonably possible to
+follow in a meeting.
+
+
+#### GitHub
+Decisions made purely on GitHub are generally intended to either be interim
+decisions that require a Plotting Session to finalize, or decisions related
+to the organization, documentation, or beurocracy of `/corporate`.
+
+Regardless of whether or not a decision can be finalized in GitHub, the method
+of deliberation is the same.
+
+1. A branch (Corporate Overlord) or fork (Member) is created from `corporate`
+against `master` or an appropriate sub-branch.
+
+2. The appropriate template file is used in the appropriate subdirectory to
+create the relevant request or proposed addition.
+
+3. A Pull Request is made against the relevant branch, with commit comments
+written using the standards described in `CONTRIBUTING.md`
+
+4. Corporate Overlords are automatically asked to review the Pull Request.
+
+5. Corporate Overlords vote using GitHub's review feature.  Approval is a yes
+vote, Rejection is a no vote, and Comments are used to request more information
+or changes.  Active abstention should be communicated via a comment review.
+
+6. If a plotting session is not required, a vote is considered passed if there
+are enough approval votes as determined by this document based on the decision
+type.  At this point the passage should be noted by the Overmind or the Secretary
+of Lore and the relevant action based on the decision can be undertaken.
+
+7. If a plotting session is required, the votes are simply an indication of favor
+and depending on the decision type may indicate temporary approval.
+
+A vote can be changed at any time until the decision has been determined to be final.
+
+Votes during plotting sessions override votes shown on Pull Requests, even if the same
+Corporate Overlord voted differently in both places.
+
+
+#### GitHub -> Plotting Session
+Some decisions are explored on GitHub but ultimately culminate in a live vote during
+a Plotting Session.  The process is the same as described in GitHub, but votes for
+open tickets are either scheduled into plotting sessions in advance or raised via
+a motion during the New Business portion of an agenda.
+
+
+### Voting Decisions
+The following decisions will require a formal input from the Hive Mind.
+
+
+- **Adding projects:** GitHub -> Plotting Session, 30% vote for preliminary approval,
```

Comment:
When we say "70% vote," we're saying "70% of the Corporate Overlords" voted in the affirmative?